src/Catalog/Registration/RegistrationMakerCatalogItem.cs

```diff
@@ -108,9 +119,7 @@ private DateTime GetPublishedDate()
 
                 if (pubTriple != null)
                 {
-                    ILiteralNode node = pubTriple.Object as ILiteralNode;
-
-                    if (node != null)
+                    if (pubTriple.Object is ILiteralNode node)
```

Comment:
I fixed a couple VS info messages while making this change--I have no problem reverting them if there's push-back.